chrome-launcher.ts

```diff
@@ -94,7 +96,7 @@ class Launcher {
   private requestedPort?: number;
   private connectionPollInterval: number;
   private maxConnectionRetries: number;
-  private chrome?: childProcess.ChildProcess;
+  public chrome?: childProcess.ChildProcess;
```

Comment:
sg. seems like we can also add private/public/public to the props on L105-107?

Reply:
oh good catch, public is actually not-needed since when omitted it is the default. I will drop it from here.